core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
 policies:
   - uid: mondoo-okta-security
     name: Okta Organization Security
-    version: 1.0.0
+    version: 2.0.0
     license: MPL-2.0
     tags:
       mondoo.com/category: security
       mondoo.com/platform: okta-org,saas
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    scoring_system: 2
     docs:
-      desc: "## Overview\n\nThe Mondoo Okta Security policy bundle provides security recommendations for Okta organizations.\n\n## Remote scans with cnspec\n\ncnspec can provide on-demand security assessments of infrastructure and services without installing agents or integrations. cnspec connects with and scans a growing number of targets. \n\nFor a complete list, run: \n\n```bash\ncnspec scan --help\n``` \n\n## The cnspec Okta provider\n\nThis policy uses the cnspec Okta provider to authenticate with Okta's API and remotely scan an Okta organization. To learn more, run: \n\n```bash\ncnspec scan okta --help\n```\n\n### Create an API token\n\nThe Okta provider for cnspec requires an API token to authenticate and run scans. To learn how to create an API token, read [Create an API token](https://developer.okta.com/docs/guides/create-an-api-token/main/) in the Okta documentation.\n\n### Configure an OKTA_TOKEN environment variable\n\nYou supply your API token to cnspec using the `OKTA_TOKEN` environment variable. \n\nOn Linux or macOS, run:\n\n```bash\nexport OKTA_TOKEN=<OKTA_TOKEN>\n```\n\nOn Windows, run:\n\n```powershell\n$Env:OKTA_TOKEN = \"<OKTA_TOKEN>\"\n```        \n\n## Scan an Okta organization\n\nTo assess the configuration of an Okta organization, run:\n\n```bash\ncnspec scan okta --organization YOURDOMAIN.okta.com --token $OKTA_TOKEN  --policy-bundle mondoo-okta-security.mql.yaml\n```\n\n## Join the community!\n\nOur goal is to build policies that are simple to deploy, accurate, and actionable. \n\nIf you have suggestions for improving this policy or if you need support, join the [Mondoo community in GitHub Discussions](https://github.com/orgs/mondoohq/discussions).\n"
+      desc: |
+        # Overview
+
+        The Mondoo Okta Security policy bundle provides security recommendations for Okta organizations. This policy supports scanning of Okta Organizations as well as Terraform projects using the [Okta Terraform provider](https://registry.terraform.io/providers/okta/okta/latest/docs) from the HashiCorp Terraform Registry.
```

Comment:
The capitalization of Organizations needs to be consistent. Seem like it should always be lowercase here.